### PR TITLE
Fix references to account DB

### PR DESCRIPTION
**Pull request by NullSoldier**
## Summary

Fixing places that still refer to to the wallet DB as account DB

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/commands/reset.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export default class Reset extends IronfishCommand {
 
     const message =
       '\nYou are about to destroy your node databases. The following directories and files will be deleted:\n' +
-      `\nAccounts: ${accountDatabasePath}` +
```
Comment (ygao76) [inline ironfish-cli/src/commands/reset.ts:57]: Can you change `Accounts` in the message?

File: ironfish/src/node.ts [hunk 2]
```diff
@@ -246,7 +246,7 @@ export class IronfishNode {
     })
 
     const accountDB = new WalletDB({
```
Comment (ygao76) [inline ironfish/src/node.ts:248]: variable name is still accountDB

File: ironfish-cli/src/commands/backup.ts [hunk 3]
```diff
@@ -97,7 +97,7 @@ export default class Backup extends IronfishCommand {
     }
 
     if (!flags.accounts) {
```
Comment (ygao76) [inline ironfish-cli/src/commands/backup.ts:99]: flag variable name is still accounts

File: ironfish-cli/src/commands/reset.ts [hunk 4]
```diff
@@ -45,7 +45,7 @@ export default class Reset extends IronfishCommand {
       this.exit(0)
```
Comment (ygao76) [inline ironfish-cli/src/commands/reset.ts:45]: the warning message still has accounts.

File: ironfish-cli/src/commands/reset.ts (hunk 1 above)
Comment (NullSoldier) [reply ironfish-cli/src/commands/reset.ts:57]: Fixed

File: ironfish-cli/src/commands/reset.ts (hunk 4 above)
Comment (NullSoldier) [reply ironfish-cli/src/commands/reset.ts:45]: Fixed

File: ironfish-cli/src/commands/backup.ts (hunk 3 above)
Comment (NullSoldier) [reply ironfish-cli/src/commands/backup.ts]: Fixed

File: ironfish/src/node.ts (hunk 2 above)
Comment (NullSoldier) [reply ironfish/src/node.ts]: Fixed

**Comment by NullSoldier** [comment]
I changed the description on the PR now since we're catching more spots. Thanks for catching these other instances of account db.

**Review by rohanjadvani — APPROVED** [review]

**Pull request merged**